docs/wiki/introduction/sql.md

```diff
@@ -269,7 +269,35 @@ The following trig functions: `sin`, `cos`, `tan`, `cot`, `asin`, `acos`, `atan`
 
 #### String functions
 
-String parsing functions are always helpful. Some are helpful within subqueries, so they make sense as local-additions:
+String functions are always helpful. Some are helpful within
+subqueries, so they make sense as local-additions:
```

Comment:
Remove this line? Grammar is a little strange and I'm not sure it adds much value.

Reply:
Sure, I haven't really been copyediting the existing docs